sim/druid/forms.go

```diff
@@ -142,7 +142,7 @@ func (druid *Druid) registerCatFormSpell() {
 			druid.AutoAttacks.SetMH(clawWeapon)
 
 			druid.PseudoStats.ThreatMultiplier *= 0.71
-			druid.PseudoStats.BaseDodge += 0.02 * float64(druid.Talents.FelineSwiftness)
+			druid.AddStatDynamic(sim, stats.Dodge, 0.02*float64(druid.Talents.FelineSwiftness))
```

Comment:
`2*float64(druid.Talents.FelineSwiftness)` here (but you'll look for stats.Dodge/stats.Parry usage anyhow ;))